### ISSUE TITLE
portal between servers

### ISSUE THREAD

**Issue by GTink911** [zorkind/Hellion-Rescue-Project#107]
**Is your feature request related to a problem? Please describe.**
No.

**Describe the solution you'd like**
A rare component allows for moving between servers. T1 takes 30m to charge, T2 takes 20m to charge, T3 takes 10m to charge, T4 takes 5m to charge. Uses all of capacitor when activated. Upon activation you can move to a different server, taking all items in the ship and modules attached.

**Describe alternatives you've considered**
None.

**Additional context**
None.

**Comment by GuiltyNeko** [comment]
What about a pre-fab station in orbit around Hellion itself that you dock to via airlock/anchor that has no/minimal interior and just some exterior slots for several CCFGs, and a fixed amount of power (and/or resources) required per module/x amount of mass. Need to supply it with CCFGs in order to transport very many, and the "charge" time would be basically as long as it takes to give it the power it needs. Should probably consume the CCFGs in the slots when you activate it. Would be cheap enough to send a single steropes, but if you want to send your whole base, it might take a few hours to fill it with enough power.

**Comment by lochshire** [comment]
I see this as something like a "jump module" that you can add to your base. Inside is a control panel and slots for "jump cores" that have to be added. Make the cores hard to find or require a lot of them, something to make it costly to move from server to server, so people aren't jumping around every hour or so.

**Comment by GTink911** [comment]
Its a rare component, like you first find it in T3 zones. I feel like having it just consume power is too easy though, as you could just bring your entire base along. I was thinking that if you tow items it requires warp cell usage?

**Comment by GuiltyNeko** [comment]
If it's a specific location in the system, then it opens up the possibility of somebody camping at the thing and hopping in your base when you teleport into the server. And if it requires having CCFGs for more taking more than a couple modules, you have to spend a bit of time & resources making/finding them. And you STILL have to provide it with the power you need. If it's something you can just install on any ship, there's not much stopping you from hopping between servers at a rapid pace. Just jump between servers until you find one where there's an arges falling into the planet. Then bring it back to your own.

**Comment by GTink911** [comment]
Exactly, camping. It needs to be hard to find, expensive on power, only partially campable, etc.

**Comment by GuiltyNeko** [comment]
Maybe have a couple of them then. One far in, one far out. One far out is a bit cheaper on power but by its nature hard to find. And maybe the things have to be repaired to both send and receive anything. Maybe if they aren't repaired in the destination server (they would start heavily damaged in every server), you could end up literally anywhere in the destination server. Making it a bit of a gamble if you don't have a friend there to tell you it's safe.
